src/Event/ImageProcessingListener.php

```diff
@@ -77,6 +77,12 @@ public function implementedEvents() {
 	 * @return boolean
 	 */
 	protected function _autoRotate($imageFile, $format) {
+		// Auto-rotate applies only to JPEG files (i.e. not to PNG, GIF, etc..)
+		$format = strtolower($format);
+		if (!in_array($format, array('jpg', 'jpeg', 'tif', 'tiff'))) {
```

Comment:
Can you please use the short syntax for the array declaration? `[]`.

I know the old syntax is still present sometimes where I haven't updated the code yet.